media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -201,15 +211,15 @@ pkg_postinst() {
 		elog
 	fi
 
-	elog "JACK emulation is incomplete and not all programs will work. PipeWire's"
```

Comment:
We shouldn't display all of this all the time when pipewire is getting merged to disk. Users tend to ignore it with the result to miss really important messages once we need to communicate them. See https://github.com/gentoo/gentoo/blob/13ec4ca6975058cf8cd810da440d1e42618b7986/sys-apps/lm-sensors/lm-sensors-3.6.0.ebuild#L172-L215 to get an idea what I mean.

Reply:
Thank you for the example, if you really insist, I will implement that but I would like to point out that not printing messages will not make them be read more. ;)